components/style-guide.css

```diff
@@ -219,3 +152,83 @@ body{
     border-radius: var(--radius-full);
 }
 
+/* TYPOGRAPHY */
+
+/* Type Scale */
+.h1{
+    font-size: 6rem;
+    font-weight: var(--light);
+    letter-spacing: -1.5px;
+}
+
+.h2{
+    font-size: 3.75rem;
+    font-weight: var(--light);
+    letter-spacing: 0.5px;
+}
+
+.h3{
+    font-size: 3rem;
+    font-weight: var(--regular);
+    letter-spacing: 0px;
+}
+
+.h4{
+    font-size: 2.125rem;
+    font-weight: var(--regular);
+    letter-spacing: 0.25px;
+}
+
+.h5{
+    font-size: 1.5rem;
+    font-weight: var(--regular);
+    letter-spacing: 0px;
+}
+
+.h6{
+    font-size: 1.25rem;
+    font-weight: var(--medium);
+    letter-spacing: 0.25px;    
+}
+
+.sub-title1{
+    font-size: 1rem;
+    font-weight: var(--regular);
+    letter-spacing: 0.15px;
+}
+
+.sub-title2{
+    font-size: 0.875rem;
```

Comment:
change this into px as it is less than 10.

Reply:
actually my base font-size is 16 px so this will be equal to 14px